source/includes/steps-install-shell-linux-rpm.yaml

```
@@ -0,0 +1,21 @@
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: navigate-dlc
+ref: navigate-dlc-linux-rpm
+---
+title: "Download the Red Hat 64-bit ``.rpm`` package."
+ref: download-archive-rhel
+level: 4 
+---
+title: "Extract and install the ``.rpm`` package."
+ref: extract-archive-rpm
+level: 4
+content: |
+  To install the |mdb-shell|, run the following command from the
+  directory containing the ``mongosh`` ``.rpm`` package:
+
+  .. code-block:: sh
+
+     sudo yum install mongosh-{+version+}-x86_64.rpm
```

```suggestion
     sudo yum install -y mongosh-{+version+}-x86_64.rpm
```

• Super minor. `yum` prompts you in the install: "are you really really sure you want to install??" For our purposes, we generally skip that prompt by passing `-y` to `yum`.

[Example in manual](https://docs.mongodb.com/manual/tutorial/install-mongodb-enterprise-on-red-hat/#install-mongodb-enterprise-version).